docs/sdk/v3/guides/02-quoting.md

```diff
@@ -1,6 +1,113 @@
 ---
 id: quoting
-title: Getting a Quote
----
+title: Quoting
+---     
+
+## Introduction
+
+This guide will cover how to get the current quotes for any token pair on Uniswap.
+It is based on the [Quoting code example](https://github.com/Uniswap/examples/tree/main/v3-sdk/quoting), found in the Uniswap code examples [repository](https://github.com/Uniswap/examples).
+
+In this example we will use `quoteExactInputSingle` to get a quote for the pair **USDC - WETH**.
+
+The guide will **cover**:
+1. Computing the **USDC - WETH** Pool's deployment address
+2. Setting up a reference to the Pool smart contract
+3. Getting Pool metadata from the Pool smart contact
+4. Setting up a reference to the Quoter contract
+5. Getting **quotes** from the Quoter contract
+
+The **inputs** to the example are:
+1. Token in
+2. Token out
+3. Amount in
+
+The **output** of the guide is a visual representation of the quote.
```

Comment:
This is covered IMO by the background / we should include this after in the "Setup Info" step

Reply:
This feels super important - it's not about the medium of the output, but rather WHAT the output.

Reply:
I agree we 100% should include it, but I think this sentence adds a lot more confusing by focusing on the "visual representation" and not really describing the output exactly. The issue here IMO is the placement/context of the section, which can probably be along the lines of:

```
This example will display a live quote for the given token pair based on the amount of the input token provided.
```

This then acts as a summary and feels more directly usable, and then the "will cover" section explains the steps overview, and then we dive in! That will then pair nicely with being capped off by the "here's the example code, follow the README setup if you'd like to run this example.

Reply:
I moved things around a bit - inputs, process, output/expectation. It reads better for me this way as it also follows the way the guide flows.

Reply:
Looks great!